content/400-reference/300-database-reference/01-database-features.mdx

```diff
@@ -133,7 +133,7 @@ The following table lists common MongoDB features and describes the level of sup
 | Arrays                                    | ✔️                  |                                                                                                                                                                                                               |
 | Enums                                     | ✔️                  | Implemented at Prisma level                                                                                                                                                                                   |
 | Native database types                     | ✔️                  | See: [Field mapping reference](/reference/api-reference/prisma-schema-reference#model-field-scalar-types)                                                                                                     |
-| JSON support                              | ✔️                  | Advanced `Json` field filtering not supported                                                                                                                                                                 |
+| JSON support                              | ✔️                  | Advanced `Json` field filtering is supported by [Postgres and MySQL since 2.26.0](https://www.prisma.io/docs/concepts/components/prisma-client/working-with-fields/working-with-json-fields#filtering-on-a-json-field)                                                                                                                                                                |
```

Comment:
```suggestion
| JSON support                              | ✔️                  | Advanced `Json` field filtering is supported by [Postgres and MySQL since 2.26.0](/concepts/components/prisma-client/working-with-fields/working-with-json-fields#filtering-on-a-json-field)                                                                                                                                                                |
```